support ariadne recommendations

linpengzhang opened the pull request:
adapt the code to the new LwJson format which now supports Ariadne recommendations

Tested on https://github.com/lacework-dev/code-analysis-test-repo/pull/307

PatGod commented on src/lw-json.ts:
```diff
@@ -2,18 +2,41 @@ export interface LWJSON {
 }
 
-// Fix suggestion
+export interface VersionEdge {
```
Could we please rename 'VersionEdge' by 'NextNode' as in lwjson.go?

PatGod commented on src/lw-json.ts:
```diff
@@ -2,18 +2,41 @@ export interface LWJSON {
-  Type?: string
-  Version?: string
+export interface VersionNode {
+  Version: string
```
Please rename 'VersionNode' by 'SortedVersionGraphNode' as in lwjson.go

PatGod commented on src/lw-json.ts:
```diff
@@ -2,18 +2,41 @@ export interface LWJSON {
+}
+
+export interface VersionGraphInfo {
+  NodeCnt?: number
```
Please rename ' VersionGraphInfo' by 'VersionGraphStruct' as in lwjson.go

PatGod commented on src/lw-json.ts:
```diff
@@ -2,18 +2,41 @@ export interface LWJSON {
+}
+
+export interface Recommendation {
+  FixVersion: FixVersion
```
Please replace 'Recommendation' by 'FixInfo' as in lwjson.go?

PatGod commented on src/lw-json.ts:
```diff
@@ -2,18 +2,41 @@ export interface LWJSON {
 }
 
 export interface Info {
-  FixVersion?: FixVersion
```
Please rename 'Info' by 'FixSuggestionInfo' as in lwjson.go

PatGod reviewed (changes requested):
Could you please perhaps address these minor cosmetic suggestions?
It would be nice to keep the same terms as used in lwjson.go, as much as possible.
Thanks!

linpengzhang commented:
> Could you please perhaps address these minor cosmetic suggestions? It would be nice to keep the same terms as used in lwjson.go, as much as possible. Thanks!

Thanks for the carefully review, I have addressed the comments and ensured the test passes again!
